examples/test-exec-doc.yml

```diff
@@ -0,0 +1,221 @@
+                    break
+            if pr is None:
+                # Create a new pull request if none exists
+                pr = repo.create_pull(title=PR_TITLE, body=PR_BODY, head=f"{owner}:{BRANCH_NAME}", base='main')
+            pr_number = pr.number
+        else:
+            pr_number = None  # No PR needed if on 'main' branch
+
+        # Get the list of changed Markdown files
+        changed_files_env = os.getenv('CHANGED_MARKDOWN_FILES', '')
+        changed_files = [f.strip() for f in changed_files_env.split('\n') if f.strip().endswith('.md')]
+
+        if not changed_files:
+            print("No Markdown files to test.")
+            exit(0)
+
+        # Initialize a list to store test results
+        results = ["**Test Results**\n\n==========\n\n"]
+        # Iterate over changed Markdown files
+        for file_path in changed_files:
+            # Execute the Innovation Engine on the Markdown file
+            result = subprocess.run(['ie', 'execute', file_path, '--environment', 'github-action'])
+            if result.returncode != 0:
+                # If execution fails, extract error log from 'ie.log'
+                error_log = ''
+                try:
+                    with open('ie.log', 'r') as log_file:
+                        lines = log_file.readlines()
+                        # Filter out error lines
+                        error_lines = [line for line in lines if "level=error" in line]
+                        error_log = ''.join(error_lines)
+                except FileNotFoundError:
+                    error_log = "ie.log not found."
+
+                # Prepare issue details
+                doc_path = file_path
+                issue_title = f"DOC FAILING TESTS: {doc_path}"
+                issue_body = f"Hey! Your executable document is not working. Please fix the errors given below.\n\nLink to Doc: https://github.com/{owner}/{repo_name}/blob/main/{doc_path}\n\n### Error Details\n\n***{error_log}***"
+                issues = repo.get_issues(state='open')
+                # Check if an issue with the same title already exists
+                existing_issue = next((issue for issue in issues if issue.title == issue_title), None)
+                if existing_issue:
+                    # Add a reminder comment to the existing issue
+                    existing_issue.create_comment(f"Reminder: Please address the issues in {doc_path}.")
+                    issue_url = existing_issue.html_url
+                    issue_number = existing_issue.number
+                else:
+                    # Create a new issue if none exists
+                    try:
+                        new_issue = repo.create_issue(title=issue_title, body=issue_body, assignee=owner)
+                    except:
+                        # Assign to default user if assigning to owner fails
+                        new_issue = repo.create_issue(title=issue_title, body=issue_body, assignee='naman-msft')
```

Comment:
You're still assigning these issues to yourself here

Reply:
updated